README.md

```diff
@@ -1,6 +1,30 @@
-# atlas-metadata-validator
+# Expression Atlas metadata validator
+
+This is a python module to parse a set of MAGE-TAB files and check for compatibility with the Expression Atlas and Single Cell Expression Atlas analysis pipelines. The main validation script automatically detects the experiment type from the MAGE-TAB and runs the respective tests. Currently general checks (for bulk and single-cell experiment) as well as Single Cell Expression Atlas specific checks are supported. 
+
+## Requirements
+
+Python 3.6 and requests (tested with version 2.20.1)
+
+
+## Install
+
+Install with pip:
+```
+pip install atlas-metadata-validator
+```
 
 ## Single-cell MAGE-TAB validator
 
-Checks to run on MAGE-TAB metadata files before loading an experiment into Single Cell Expression Atlas. The checks are mainly covering the pre-validation by https://github.com/ebi-gene-expression-group/scxa-control-workflow/blob/master/bin/sdrfToNfConf.R in order to guarantee correct processing.
+A MAGE-TAB pre-validation module for running checks that guarantee the experiment can be processed for [Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/home). The checks are mainly covering the pre-validation by https://github.com/ebi-gene-expression-group/scxa-control-workflow/blob/master/bin/sdrfToNfConf.R in order to guarantee correct processing. It reads metadata directly from the MAGE-TAB and generates a log file in the directory of the IDF file.
+
+The checks can be invoked using the atlas_validation script with an IDF file path as input:
+```
+python atlas_validation.py path/to/test.idf.txt 
```

Comment:
I noticed the 'python' bit of this is unnecessary with the pip install

Reply:
Yes, that's right. Fixed.